packages/react/src/components/Web3Modal.tsx

```diff
@@ -14,7 +14,7 @@ interface Props extends Omit<ConfigCtrlState, 'enableStandaloneMode' | 'standalo
 /**
  * Component
  */
-export function Web3Modal({ ethereumClient, ...config }: Props) {
+function CreateWeb3Modal({ ethereumClient, ...config }: Props) {
```

Comment:
NB: Why was this renamed?

Reply:
to export it as default `Web3Modal` via memo here (names were clashing) https://github.com/WalletConnect/web3modal/pull/845/files/62d84f0a4816ac325ae1dac83391ee2c3120280a#diff-3477080de4f7c2140710399a4ede9295341ea1680f353b105b0558095da90132R34